validator/client/validation_client.go

```diff
@@ -267,3 +290,131 @@ func (r *ExecutionClientRun) Close() {
 		}
 	})
 }
+
+// BoldValidationClient implements bold validation client through redis streams.
+type BoldValidationClient struct {
```

Comment:
this entire unit should be under validator/client/redis 
It can include a redis.ValidatioClient but should not depend on the normal ValidationClient